examples/LaTeX Circuits.ipynb

```diff
@@ -0,0 +1,204 @@
+      "text/plain": [
+       "<IPython.core.display.Image object>"
+      ]
+     },
+     "execution_count": 9,
+     "metadata": {
+      "image/png": {
+       "width": 300
+      }
+     },
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# extra kwargs are passed are passed straight to `IPython.display.Image`\n",
+    "display(prog, width=300)"
```

Comment:
this gives me an obscure error:

```
---------------------------------------------------------------------------
RuntimeError                              Traceback (most recent call last)
<ipython-input-3-6f822dbd0b9c> in <module>
      1 # extra kwargs are passed are passed straight to `IPython.display.Image`
----> 2 display(prog, width=300)

~/code/rigetti/github/pyquil/pyquil/latex/ipython.py in display(circuit, settings, **image_options)
     40         result = subprocess.call([pdflatex_path, "-output-directory", tmpdir, texfile.name])
     41         if result != 0:
---> 42             raise RuntimeError("pdflatex error")
     43 
     44         png = os.path.join(tmpdir, 'diagram.png')

RuntimeError: pdflatex error
```

which I see in my jupyter notebook logs is actually:

```
! Package tikz Error: I did not find the tikz library 'quantikz'. I looked for
files named tikzlibraryquantikz.code.tex and pgflibraryquantikz.code.tex, but n
either could be found in the current texmf trees..
```

Reply:
we should add some good documentation for installing all the requirements for this module

Reply:
Adding sphinx docs, and also adding a bit more care to the error reporting here.